vowpalwabbit/learner.h

```diff
@@ -895,6 +925,22 @@ struct common_learner_builder
     _learner->persist_metrics_fd.base = _learner->learn_fd.base;
     return *static_cast<FluentBuilderT*>(this);
   }
+
+  FluentBuilderT& set_copy(void (*fn_ptr)(DataT&, size_t& from, size_t& to))
```

Comment:
this name is super vague. Can you please update this https://github.com/VowpalWabbit/vowpal_wabbit/wiki/What-is-a-learner%3F ?

Reply:
The point of this function is a way for an upper reduction like automl to signal a reduction below to copy the state from one offset to another. I'll document it.

Reply:
Are `from` and `to` in_out parameters? Can you also rename then to something like `from_offset`